docs/guides/execution-modes.mdx

```diff
@@ -5,73 +5,169 @@ description: An overview of the available execution modes in Qiskit Runtime; ses
+   <Admonition type="note">
+   The job must go through the normal queue to reactivate the workload.
+   </Admonition>
+3. If the TTL value is reached, the workload ends and any remaining queued jobs fail. However, any jobs already running will run to completion.
 
-[**Session mode**](sessions): A dedicated window for running a multi-job workload. This allows users to experiment with variational algorithms in a more predictable way and even run multiple experiments simultaneously, taking advantage of parallelism in the stack. Use sessions for iterative workloads or experiments that require dedicated access.  To run in session mode, specify `mode=session` when instantiating a primitive, or run the job in a session context manager. Learn more about sessions in the [Introduction to sessions,](/guides/sessions) and see [Run jobs in a session](run-jobs-session) for examples.
+The following video illustrates the session basic workflow, using sessions as an example:
+
+<video title="A user starts a session job and becomes the priority user.  They submit jobs to the QPU while other users wait. After the prioritized user's session finishes, the next person's jobs can begin processing." className="max-w-auto h-auto" controls>
+  <source src="/videos/guides/sessions/demo.mp4" type="video/mp4"></source>
+</video>
 
 <span id="sessions-versus-batch-usage" />
 
-## Choose batch or sessions mode
+## Choose the right execution mode
+
+Utility-scale workloads can take many hours to complete, so it is important that both the classical and quantum resources are scheduled efficiently to streamline the execution. Execution modes provide flexibility in balancing the cost and time tradeoff to use resources optimally for your workloads. There are several aspects to consider when choosing which execution mode to use, such as _usage_, overall execution time (TTL), and time between jobs (ITTL).
+
+The the benefits of each are summarazed below:
+
+- **Batch**
+   - The entire batch of jobs is scheduled together and there is no additional queuing time for each.
+   - The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially.
+   - There is usually minimal delay between jobs, which can help avoid drift.
+   - If you partition your workload into multiple jobs and run them in batch mode, you can get results from individual jobs, which makes them more flexible to work with. For example, if a job's results don't meet your expectations, you can cancel the remaining jobs. Also, if one job fails, you can re-submit it instead of re-running the entire workload.
+- **Session**
+   - Dedicated and exclusive access to the QPU during the session active window.
+   - Can run multiple experiments simultaneously.
```

Comment:
You can technically run multiple experiments simultaneously in batch as well, as long as they are not iterative workloads. 

Instead of this line, perhaps we can say "all the benefits you get from batch mode". Multiple people have asked me if parallel compilation only happens in batch and not session, since it's not explicitly listed under session.